Added Nebraska scraper

rnelson opened the pull request:
Adds a scraper to pull commodity, service, and agency RFPs from the Nebraska State Purchasing Bureau. Initial work by @rnelson and @dlipskey.

rnelson committed:
Added Nebraska scraper

Initial work by @rnelson and @dlipskey

ajb commented:
Awesome work! Do you mind taking a look at this when you have a sec?

![screen shot 2014-03-17 at 10 37 57 am](https://f.cloud.github.com/assets/1270317/2436948/db6de156-ade1-11e3-930f-07532b1b4f0f.png)

rnelson committed:
Fixed the NE scraper as of 2014-03-17

CSS style on some table cells changed, which broke the parser. That
particular issue is now fixed as we're looking for the nth child <td>
instead of the nth child node with .cell-purch.

rnelson commented:
Fixed in opennebraska/openrfps-scrapers@610c56a. These pages are all done by hand in DreamWeaver and someone didn't set the correct style on the text, which broke the parser. I've updated all of the code to look for a &lt;td&gt; instead of the specific style.

ajb commented:
Thanks again, merged! :stars:
